api/can_api_definition.py

```diff
@@ -79,6 +79,10 @@ class _Actuals(pydantic.BaseModel):
     )
     hospitalBeds: Optional[_ResourceUtilization] = pydantic.Field(...)
     ICUBeds: Optional[_ResourceUtilization] = pydantic.Field(...)
+    # contactTracers count is available for states, not counties.
+    contactTracers: Optional[int] = pydantic.Field(
```

Comment:
You'll also have to add this into the place where actuals get set.  see [here](https://github.com/covid-projections/covid-data-model/blob/master/libs/functions/generate_api.py#L89)

Reply:
Done, thank You for the pointer.